assets/js/modules/search-console/components/module/ModuleOverviewWidget/index.js

```diff
@@ -87,6 +89,8 @@ const ModuleOverviewWidget = ( { Widget, WidgetReportZero, WidgetReportError } )
 		);
 	}
 
+	const { compareRange, currentRange } = partitionReport( data, { rangeLength: dateRangeLength } );
```

Comment:
Since this is only needed in the `Stats` component, why not do it there?

Reply:
I originally did but thought it might be cleaner to do this way since it needed the date range as well which it would need another prop for. I guess it really wasn't much of an improvement so I've moved it back into `Stats` with the added prop rather than changing the props it receives data from.